### PR TITLE
Completed Wishlist component

### PR DESCRIPTION
**Pull request by varun1233**

### CODE REVIEW

File: pages/wishlist/wishlist.html [hunk 1]
```diff
@@ -0,0 +1,205 @@
...
+                <span class="card-icon">
+                  <i class="fas fa-heart"></i>
+                </span>
+              </h3>
+              <p class="product-description font-weight-regular">
+                Gaming Chair Adjustable Back. Red, Black, DIY(Do-It-Yourself)
+              </p>
+              <p class="product-pricing">
+                <b>₹12000</b>
+                <span class="strike-through font-weight-regular">₹1200</span>
```
Comment (amantulsyan35) [inline pages/wishlist/wishlist.html:66]: can use p tag instead off span tags makes it more semantic

**Comment by kaustubh-jsr** [comment]
The UI looks professional. Awesome work.Looks good to merge.

**Review by zoef-yt — APPROVED** [review]
The code looks good. UI is also looking good. good to merge.

**Review by Sahil25061999 — APPROVED** [review]
code looks good, good work using target for a tags, a ui suggestion to remove hover from the card and instead give hover effect to the buttons inside the card.

File: pages/wishlist/wishlist.html (hunk 1 above)
Comment (varun1233) [reply pages/wishlist/wishlist.html:66]: Okay will look into it. Thanks for the suggestion

File: pages/wishlist/wishlist.html [hunk 2]
```diff
@@ -0,0 +1,205 @@
...
+  </head>
+  <body>
+    <!-- Header -->
+    <nav class="navbar">
+      <div class="logo font-weight-black align-items-center h2">
+        <a
+          class="display-flex font-weight-black align-items-center h2"
+          href="/index.html"
+        >
+          <span class="site-name">Apex &nbsp;</span><span>Furniture</span>
```
Comment (gouravm20) [inline pages/wishlist/wishlist.html:18]: everything is fine 

but  span mainly use to --> display out of box design ==> different from all 
using span twice together  may be fix

**Review by gouravm20 — COMMENTED** [review]
website look wise look great

File: pages/wishlist/wishlist.html (hunk 2 above)
Comment (gouravmarch20) [inline pages/wishlist/wishlist.html:18]: everything is fine

but span mainly use to --> display out of box design 
using span twice together may be fix but it also fine

**Review by gouravmarch20 — COMMENTED** [review]
everything is fine in ui side

**Pull request merged**